instance/tests/integration/ansible/playbooks/failignore.yml

```diff
@@ -12,6 +12,17 @@
 
 - name: Fail and ignore on purpose
   hosts: all
+  become: true
   tasks:
+    - file:
+        path: /edx/bin/
+        state: directory
+        mode: '0755'
+    - copy:
```

Comment:
This was added because there's no real `/edx/bin/supervisorctl` in the test instance, as each of the four tests in this test group create an independent instance to prevent conflicts.